flixel/FlxSprite.hx

```diff
@@ -1627,6 +1633,12 @@ class FlxSprite extends FlxObject
 		}
 		return doFlipY;
 	}
+
+	@:noCompletion
+	inline function get_graphicWidth():Dynamic
```

Comment:
```suggestion
	inline function get_graphicWidth():Float
```

Reply:
thanks, I couldn't think of what to change that to

Reply:
I'll change it whenever I get up after sleeping

Reply:
Cuz right now, I just wanna sleep